test/runtests.jl

```diff
@@ -1,85 +1,25 @@
 using Survival
-using Base.Test
+if VERSION >= v"0.7.0-DEV.2004"
```

Comment:
I recommended using `Compat.Test` in Distributions, so... :-)

Reply:
HAH yeah. That's indeed a better idea.